apps/fluent-tester/src/FluentTester/theme/ThemePickers.tsx

```diff
@@ -41,8 +53,24 @@ export const PartPicker: React.FunctionComponent<PartPickerProps> = (props: Part
     },
     [setValue, onChange],
   );
-  return (
-    <Picker selectedValue={value} style={themePickerStyles.dropdown} onValueChange={onValueChange}>
+
+  const themedDropdownStyles = getThemedDropdownStyles(useTheme());
+  const dropdownIconColor = useTheme().colors.buttonIcon;
```

Comment:
```javascript
const theme = useTheme();
const themedDropdownStyles = getThemedDropdownStyles(theme);
const dropdownIconColor = theme.colors.buttonIcon;
```
Especially with hooks, I think the number of times you call the hook matters, as it uses order of calls to determine what to return.

Reply:
I did this but did not push it because I got confused 😅